docs/addon-extensions/amp-rules-configurator-addon.md

```diff
@@ -0,0 +1,39 @@
+# AMP Rules Configurator Add-on
```

Comment:
Is this doc page obselete if you are dependent on the framework for this change?

Reply:
correct, removed